OpenRA.Game/Game.cs

```diff
@@ -59,6 +59,11 @@ public static class Game
 		static Task discoverNat;
 		static bool takeScreenshot = false;
 
+		public enum GameState { NONE, MISSIONS, SKIRMISH, MULTIPLAYER, MAP_EDITOR, REPLAYS }
```

Comment:
The engine doesn't impose any specific main menu layout (this is up to the individual mods to defined), so these can't be defined here.

Reply:
I wasn't really sure where to put LastGameState since it needed to outlive the MainMenuLogic instance. Could it maybe live in ModData somewhere in a string value dictionary? Like maybe something like this could be used in MainMenuLogic:

```csharp
var lastGameState = modData.getModSpecificValue("LastGameState")
```
Then this way mods that have a different layout can handle this on their own.

Reply:
Making it static will share it across all MainMenuLogic instances.